Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -149,7 +149,11 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
    ```bash
```

Comment:
Like above, can you replace `.bashrc` with `~/.bashrc` and drop the askubuntu link? Also, please make clear that this change only applies if one is using bash (e.g. add "If you are using Bash, ...")

Reply:
Done, kindly review it.